configure.ac

```diff
@@ -6328,6 +6335,7 @@ if test "x$ac_system" = "xLinux"; then
   plugin_fhcount="yes"
   plugin_fscache="yes"
   plugin_hugepages="yes"
+  plugin_mdevents="yes"
```

Comment:
This variable is not used. Check for `enable_mdevents == "yes"` and set `plugin_mdevents` basing on that condition.